osu.Game/Overlays/AccountCreation/ScreenEntry.cs

```diff
@@ -138,18 +138,12 @@ private void load(OsuColour colours, IAPIProvider api, GameHost host)
             passwordTextBox.Current.ValueChanged += password => { characterCheckText.ForEach(s => s.Colour = password.NewValue.Length == 0 ? Color4.White : Interpolation.ValueAt(password.NewValue.Length, Color4.OrangeRed, Color4.YellowGreen, 0, 8, Easing.In)); };
         }
 
-        protected override void Update()
-        {
-            base.Update();
-
-            if (host?.OnScreenKeyboardOverlapsGameWindow != true && !textboxes.Any(t => t.HasFocus))
```

Comment:
The `host?.OnScreenKeyboardOverlapsGameWindow != true` check seems to have been a mobile usability improvement (see #4179 and 23d0399) - did you check that this doesn't regress behaviour?